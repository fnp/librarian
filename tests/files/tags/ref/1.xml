<akap>W roku 1872 dom pod numerem 7 przy Saville Row<ref href="https://www.wikidata.org/wiki/Q1631609"></ref>, Burlington Gardens, w którym w roku 1814 zmarł Sheridan<ref href="https://www.wikidata.org/wiki/Q352725"></ref>, był zamieszkany przez Phileasa Fogga, członka londyńskiego klubu ,,Reforma"<ref href="https://www.wikidata.org/wiki/Q962177"></ref>, osobistość wybitną i wielce oryginalną.</akap>
