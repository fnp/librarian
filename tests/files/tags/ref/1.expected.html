<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">W roku 1872 dom pod numerem 7 przy Saville Row<a class="reference" data-uri="https://www.wikidata.org/wiki/Q1631609"></a>, Burlington Gardens, w którym w roku 1814 zmarł Sheridan<a class="reference" data-uri="https://www.wikidata.org/wiki/Q352725"></a>, był zamieszkany przez Phileasa Fogga, członka londyńskiego klubu „Reforma”<a class="reference" data-uri="https://www.wikidata.org/wiki/Q962177"></a>, osobistość wybitną i wielce oryginalną.</p>
</div>
